### ISSUE TITLE
Image not showing unless width/height is declared

### ISSUE THREAD

**Issue by dloused** [nuxt/image#72]
As per docs,

_When you define an image with src props, Nuxt automatically calculates aspect ratio of the image and set size of the image based on the aspect ratio. The image's width will be equal to width of parent element in DOM and its height calculates based on the aspect ratio._

However unless width and height are defined, just setting

```
<template>
  <nuxt-image src="/nuxt-icon.png" />
</template>
```

 image rendered size is 0x0

**Comment by farnabaz** [comment]
@dloused Thank you for creating issue. This module is still a work in progress and the documantation and latest release does not match.  
> When you define an image with src props, Nuxt automatically calculates aspect ratio of the image and set size of the image based on the aspect ratio. The image's width will be equal to width of parent element in DOM and its height calculates based on the aspect ratio. 

This feature will be available in next beta release (v0.0.4). It will be out in two days. ✨

**Comment by farnabaz** [comment]
This is fixed in v0.4.0  ✨

**Comment by flapili** [comment]
> This is fixed in v0.4.0 ✨

wouldn't you say 0.0.4 ?
